version/rawversion/version.go

```diff
@@ -1,5 +1,5 @@
 package rawversion
-11;rgb:0000/0000/0000
```

Comment:
Nit: this change in the second-last commit looks wrong. Can you squash this if possible?